src/backend/tools/retrieval/nyc_service_request_retriever.py

```diff
@@ -0,0 +1,105 @@
+import json
+from typing import Any, Iterable, List, Dict
+from elasticsearch import Elasticsearch
+from backend.tools.retrieval.base import BaseRetrieval
+import os
+from half_json.core import JSONFixer
+
+from backend.chat.custom.model_deployments.deployment import get_deployment
+from backend.schemas.cohere_chat import CohereChatRequest
+from backend.services.logger import get_logger
+
+cohere_api_key = os.environ.get("COHERE_API_KEY")
+logger = get_logger()
+
+
+class NYCServiceRequestRetriever(BaseRetrieval):
```

Comment:
It looks like this retriever tool isn't present in the dict mapping, should we add it? Or was this for testing?